src-electron/db/query-feature.js

```diff
@@ -31,6 +31,9 @@ const dbMapping = require('./db-mapping.js')
  * with associated device type and cluster details
  */
 async function getFeaturesByDeviceTypeRefs(db, deviceTypeRefs) {
+  if (!Array.isArray(deviceTypeRefs)) {
+    deviceTypeRefs = [deviceTypeRefs]
+  }
   let deviceTypeRefsSql = deviceTypeRefs.map(() => '?').join(', ')
```

Comment:
This change is fine for the function itself, but we might want to figure out why it is not an array only in the case of rootNode

Reply:
I agree, will figure out cause before making any decisions

Reply:
fixed, thank you @ethanzhouyc